### ISSUE TITLE
Android App Crashes after closing form with VisibleAndOverlap

### ISSUE THREAD

**Issue by AdriaanBos** [viniciusfbb/fmx_tutorials#10]
After closing a dynamic Form with Self.Close and setting `Action := caFree` on the OnClose event the app would crash with a segment fault here:
```pascal
function TipFormSystemBars.GetVisibility: TVisibilityMode;
begin
  Result := FForm.SystemStatusBar.Visibility;
end;
```
This can be fixed by adding a check to see if it's nil like so:
```pascal
function TipFormSystemBars.GetVisibility: TVisibilityMode;
begin
  if Self <> nil then
    Result := FForm.SystemStatusBar.Visibility;
end;
```

EDIT: Provided more information and a fix

**Comment by viniciusfbb** [comment]
Hi @AdriaanBos, thank you very much for reporting the issue!

I'll confirm it in the next few days, but it seems to be a RAD Studio bug: https://quality.embarcadero.com/plugins/servlet/mobile#issue/RSP-33859 Luckily it was fixed in RAD Studio 11.1.

**Comment by AdriaanBos** [comment]
Hi,

They seem to have just closed it and marked it as not reproducible. I'll also test and confirm once I have installed 11.1
